Jessie Smith/assignments/hello_world.py

```diff
@@ -0,0 +1,11 @@
+for i in range(0, 1000):
+	if i%2 != 0:
+		print i 
+
+for i in range(5, 1000000):
```

Comment:
There's also another trick you can use; if you use a third parameter in range() it is the step. i.e. range(5,100000,5)